<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBook</class>
 <widget class="QWidget" name="NewBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Библиотека: поступление книг</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(70, 70, 70)</string>
  </property>
  <widget class="QRadioButton" name="rb_form">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ручное добавление данных</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_file">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>file</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Выбрать файл с данными </string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите название книги:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 176, 176);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_author">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите имя автора:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_author">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 176, 176);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_year">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите год написания/перевода книги:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_year">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 176, 176);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Выберите жанры книги:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Биография</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Детектив</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Драма</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Исторический</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SplitHCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Комедия</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Научный</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Триллер</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ужасы</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Фантастика</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_10">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Фэнтези</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_number">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите количество одинаковых книги:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_number">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 176, 176);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_shelf">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите номер стеллажа, где будут хранится книги:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_shelf">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 176, 176);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_form_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 153, 153);</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>650</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 153, 153)</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_year">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>265</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_number">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>455</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_directory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="accessibleName">
    <string>file</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите путь к файлу:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_directory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>file</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 176, 176);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_success">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="accessibleName">
    <string>file_form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>Операция выполнена успешно.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_file_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string>file</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 153, 153)</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_type">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string>Укажите хотя бы один жанр.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_name">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>105</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_author">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>185</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_shelf">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>535</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>form</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_error">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>600</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>271</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Добавленные id книг</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
